src/NHSD.BuyingCatalogue.Identity.Api/Services/ProfileService.cs

```diff
@@ -52,9 +55,16 @@ public async Task GetProfileDataAsync(ProfileDataRequestContext context)
             }
 
             ApplicationUser user = await GetApplicationUserAsync(context.Subject);
+
             if (user is object)
             {
-                context.IssuedClaims = GetClaimsFromUser(user);
+                var claims =  GetClaimsFromUser(user);
+                var organisationName = (await _organisationRepository.GetByIdAsync(user.PrimaryOrganisationId))?.Name;
```

Comment:
Could this be refactored into its own method? e.g. GetPrimaryOrganisationName? This could potentially make it easier to read & provide some additional context to the developer reading the code. The method would then have the single responsibility of returning the primary organisation name claim.